### DISCUSSION TITLE
How to access host name on ssr(NUXT 3) ?

### DISCUSSION THREAD

**Discussion by nazariisliusarenko** [nuxt/nuxt#16032] [Questions] [answered]
How to access host name on ssr (NUXT 3)?

i achieved it like this, but is there a better way to find it ? (without rely on index in array)

const event = useRequestEvent();
  let host = "";
  if (process.server) {
    host = event.req.rawHeaders[event.req.rawHeaders.findIndex((e: string) => e === 'host') + 1];
  } else {
    host = window.location.host;
  }

**Comment by tintin10q** [comment, marked as answer]
You can do this `const host = event.node.req.headers.host;`

**Comment by nazariisliusarenko** [comment]
thanks, it works)
